### ISSUE TITLE
Blaze Powder Texture Is Messed Up After Updating To Latest Of All CoFH Mods

### ISSUE THREAD

**Issue by RoloEdits** [CoFH/Feedback#650]
after updating core, expansion,foundation,redstone arsenal,dynamics my blaze powder texture looks wonky 

![2015-07-06_23 08 26](https://cloud.githubusercontent.com/assets/12489689/8539781/43f06d56-2434-11e5-9002-bc6874577e62.png)

![2015-07-06_23 08 28](https://cloud.githubusercontent.com/assets/12489689/8539783/483250fa-2434-11e5-909c-d0e29a97d209.png)

**Comment by KingLemming** [comment]
We need a modlist to know what the actual cause might be. TF replaces the Blaze powder icon; you can disable this in the config.

Ultimately, this works fine for 99% of everyone, so you've got something screwing this up. :)

**Comment by RoloEdits** [comment]
yah i probably guessed it was on my end after learning its supposed to be animated :p

thanks for your quick response i appreciate it very much

**Comment by RoloEdits** [comment]
im posting his image just in case anyone else gets this texture error, it might provide more info

![2015-07-06_23 32 40](https://cloud.githubusercontent.com/assets/12489689/8540130/7ea67122-2437-11e5-9148-4023f8fbf370.png)

**Comment by RoloEdits** [comment]
it seems to be made up of different icons

**Comment by RoloEdits** [comment]
here is a closer look via changing my fov

![2015-07-06_23 36 09](https://cloud.githubusercontent.com/assets/12489689/8540171/e0829858-2437-11e5-92d5-ab73c4182a8e.png)

**Comment by Revemohl** [comment]
Mine doesn't show up at all (it's a transparent square), but I guess I'll just blame it on Optifine. No worries if I can change it back to the usual through the config.

**Comment by RoloEdits** [comment]
i should have also noted i'm using fastcraft by player

**Comment by KingLemming** [comment]
Just going to close this down. The config option works, and there's no great way for us to debug Fastcraft and/or Optifine.

**Comment by b0bst3r** [comment]
It's not optifine or fastcraft, I got this updating to the latest versions, so came hunting around.

At first I noticed this in the startup log

[Client thread/ERROR] [TEXTURE ERRORS]:     The missing resources for domain thermalfoundation are:
[Client thread/ERROR] [TEXTURE ERRORS]:       textures/gui/items/material/DustBlaze.png
[Client thread/ERROR] [TEXTURE ERRORS]:       textures/blocks/material/DustBlaze.png

So I made those files up as the paths and files don't exist in thermal foundation assets folder, however this did not fix the problem. 

I then went and changed the config option     B:BlazePowder=true  to false and guess what, I now get the animated version which is from Thermal Foundation (I know this because I use a resource pack and it's not the usual one I see).  I think you might have something back to front.
